### DISCUSSION TITLE
Slider color modification

### DISCUSSION THREAD

**Discussion by csiyer** [jspsych/jsPsych#1959] [Q&A]
Hi there, 
I'm pretty new to jsPsych and JavaScript/CSS in general. I'm designing an experiment using a range slider using 6.3.1, where the new custom slider has no sliding background color (the older default on Chrome was the one with blue sliding from the left side). I like the new slider much more (the custom jsPsych one), but what I need is to make the slider background color change _from the middle_ (rather than from the left side, like the old default did). 

In other words, sliding the thumb to the either side reveals blue-highlighted track from the slider's center to the thumb. It's probably a pretty simple thing, but I can't figure out how to do it. Thanks!
